src/query/config/m3coordinator-local-etcd.yml

```diff
@@ -28,10 +28,6 @@ clusters:
             - zone: embedded
               endpoints:
                 - 127.0.0.1:2379
-        seedNodes:
```

Comment:
Why are you removing this? Dont you need it to setup the embedded etcd?

Reply:
Not for the coordinator (this is a coordinator config)